app.py

```diff
@@ -17,17 +17,21 @@ def do_GET(self):
             result = {}
             encountered_issues = False
             for service in services:
-                service_url = services[service]
-                logging.info('calling for client ' + service + ': ' + service_url)
-                request = urllib.request.Request(service_url)
-                try:
-                    with urllib.request.urlopen(request) as response:
-                        data = json.loads(response.read().decode('UTF-8'))
-                        result[service] = data
-                except URLError:
-                    logging.error('service not available ' + service_url, exc_info=True)
-                    encountered_issues = True
-                    result[service] = 'unavailable'
+                static_version, _value = services[service]
+                if static_version:
```

Comment:
Puh ich bin mir nicht sicher ob ich den weiteren if check in Kombination mit unseren deployments richtig verstehe. Die "Readability" verbessert er auf jedenfall nicht. Formal sieht aber alles richtig aus.

Reply:
true, static just means won't change and that its supplied from an env var rather from i need to do an additional http call